test/test_math.c

```diff
@@ -0,0 +1,56 @@
+#include <CUnit/CUnit.h>
```

Comment:
Two ways you could extend this 

1. Separate file per operator
2. Separate suite per operator, and separate files for other functionality (maybe validation of user input is the only other function to test in this project)